src/app/containers/CpsFeaturesAnalysis/index.test.jsx

```diff
@@ -99,6 +99,19 @@ jest.mock('#lib/utilities/isLive', () => jest.fn());
 describe('CpsFeaturesAnalysis', () => {
   beforeEach(() => {
     jest.resetAllMocks();
+    Object.defineProperty(window, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation(query => ({
+        matches: false,
+        media: query,
+        onchange: null,
+        addListener: jest.fn(), // deprecated
+        removeListener: jest.fn(), // deprecated
+        addEventListener: jest.fn(),
+        removeEventListener: jest.fn(),
+        dispatchEvent: jest.fn(),
+      })),
+    });
```

Comment:
Curious why this is needed?

Reply:
https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom

The `useMediaQuery()` uses `window.matchMedia` which isn't implemented in jsdom so we have to mock it to stop the implementation from erroring. 

A bit confused about it though because the navigation container uses the `useMediaQuery` but in the unit tests for those they don't mock it for some reason, and they still run fine, but I haven't really dug that deep into it.

Reply:
Ah okay 👍  it's already being implemented here: https://github.com/bbc/simorgh/blob/850702f346153597aabbbca9b6d551cc315702de/src/testHelpers/jest-setup.js#L17

but for some reason it's being implemented as a mock function, so I guess the `jest.resetAllMocks();` in this `beforeEach` is messing with that